src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx

```diff
@@ -206,6 +208,19 @@ describe("polling to verify bid position", () => {
         })
       )
     })
+    it("updates the main auction screen", () => {
+      const component = renderer.create(<ConfirmBid {...initialProps} navigator={mockNavigator} />)
+      component.root.instance.setState({ conditionsOfSaleChecked: true })
+      mockphysics.mockReturnValueOnce(Promise.resolve(mockRequestResponses.pollingForBid.reserveNotMet))
+      relay.commitMutation = jest.fn((_, { onCompleted }) => {
+        onCompleted(mockRequestResponses.placeingBid.bidAccepted)
+      })
+
+      component.root.findByType(Button).instance.props.onPress()
+      jest.runAllTicks()
+
+      expect(mockPostNotificationName).toHaveBeenCalledTimes(2)
```

Comment:
Nice – I think either adding a comment explaining the `2`, or having two `expect()`'s for two sets of parameters, could help clarify to this. Really nice test.

Reply:
Right- I first wanted to check that I could assert anything at all.